dashboard/src/components/LoadingWrapper/LoadingWrapper.v2.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+
+import Spinner from "../js/Spinner";
+
+export interface ILoadingWrapperProps {
+  loaded?: boolean;
+  size?: string;
+  children?: any;
+}
+
+function LoadingWrapper(props: ILoadingWrapperProps) {
+  return props.loaded ? (
+    props.children
+  ) : (
+    <Spinner medium={props.size === "medium"} small={props.size === "small"} />
```

Comment:
You can keep `medium` and `small` props. It's more readable when using them:

```
<LoadingWrapper medium />
```

In this way, you don't need to go to the implementation to see the available sizes in the JSX.

Reply:
was thinking on maintaining the current API but yes, it's easier to set the size that way